weblate/trans/views/basic.py

```diff
@@ -756,13 +757,21 @@ def new_language(request: AuthenticatedHttpRequest, path):
                             ),
                         )
                 try:
-                    if added and not obj.create_translations(request=request):
-                        messages.warning(
+                    if added and not obj.create_translations(
+                        request=request, run_async=True
+                    ):
+                        messages.success(
                             request,
                             gettext(
-                                "The translation will be updated in the background."
+                                "All languages have been added, updates of translations are in progress."
                             ),
                         )
+                        result = "{}?info=1".format(
+                            reverse(
+                                "component_progress",
+                                kwargs={"path": result.get_url_path()},
+                            )
```

Comment:
This redirect breaks for project and category scopes, see https://github.com/WeblateOrg/weblate/issues/12413